boards/pinetime/Makefile.features

```diff
@@ -0,0 +1,7 @@
+CPU_MODEL = nrf52832xxaa
+
+# Put defined MCU peripherals here (in alphabetical order)
+#FEATURES_PROVIDED += periph_i2c
+#FEATURES_PROVIDED += periph_uart
```

Comment:
UART is not available on the board right?

Reply:
Isn't there a debug pin or two that could be repurposed? but yeah, not really.